src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,134 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_CAPACITY = 10;
+    public static final double DEFAULT_GROW_RATE = 1.5;
```

Comment:
```suggestion
    public static final double GROW_RATE = 1.5;
```

Reply:
capacities can be different - that's why we have a default one, but there ever will be only one grow_rate